src/style.css

```
@@ -1 +1,25 @@
-/* styles go here */
+
+form :first-child {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+fieldset {
+  
+  margin-top: 20px;
+  margin-bottom: 20px;
+  box-sizing: border-box;
+}
```

Краще використовуй класи 😉

```suggestion
fieldset {
  margin-top: 20px;
  margin-bottom: 20px;
  box-sizing: border-box;
}
```